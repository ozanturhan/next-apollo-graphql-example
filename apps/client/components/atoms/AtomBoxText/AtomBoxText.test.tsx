import React from 'react';
import 'jest-styled-components';
import { render } from '@testing-library/react';
import { AtomBoxText } from '@components';

it('renders correctly', () => {
  const tree = render(<AtomBoxText />).container;
  expect(tree).toMatchSnapshot();
});
