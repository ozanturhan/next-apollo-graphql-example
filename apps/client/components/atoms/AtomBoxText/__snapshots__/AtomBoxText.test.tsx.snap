// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  font-size: 1.7rem;
  font-weight: bold;
}

@media only screen and (max-width:568px) {
  .c0 {
    font-size: 1.4rem;
  }
}

<div>
  <span
    class="c0"
  />
</div>
`;
